packages/swap/src/providers/okx/index.ts

```
@@ -0,0 +1,927 @@
+              logger.info(
+                `  - Returning unmodified OKX transaction to prevent corruption`,
+              );
+              return result;
+            } catch (modifyError) {
+              // Fallback to original transaction
+              const reserializedBuffer = transaction.serialize();
+              const result = Buffer.from(reserializedBuffer).toString("base64");
+              return result;
+            }
+          } catch (versionedError) {
+            throw versionedError; // Skip legacy fallback, it's broken
```

Biome (1.9.4)

[error] The catch clause that only rethrows the original error is useless.

An unnecessary catch clause can be confusing.
Unsafe fix: Remove the try/catch clause.


(lint/complexity/noUselessCatch)